kuma/core/whitenoise/KumaDjangoWhiteNoise.py

```diff
@@ -0,0 +1,15 @@
+from whitenoise.django import DjangoWhiteNoise
+
+
+class KumaDjangoWhiteNoise(DjangowhiteNoise):
+
+    # Custom MIME types
+    KUMA_MIMETYPES = (
+            ('application/font-woff', '.woff'),
+            ('application/font-woff2', '.woff2'),)
+
+    # Use our custom MIME types, together with the default MIME types, in
+    # WhiteNoise. When our MIME types conflict with the defaults, use ours.
+    EXTRA_MIMETYPES = tuple({e[1]: e for e in reversed(
+            KUMA_MIMETYPES + DjangoWhiteNoise.EXTRA_MIMETYPES
+        )}.values())
```

Comment:
I'm sure there there's a better way to do this. But it works, and it illustrates what I'm going for.